app/konnect/runtime-manager/kic.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Kubernetes Ingress Controller Integration
```

Comment:
❗ Reviewers: I don't love the name of this. As I was working on this and reading the brief, I didn't think "KIC in Konnect" was the best title because I thought some customers might be thinking more along the lines of "I use Kubernetes, how do I get my stuff into Konnect?" vs. "I want to add KIC stuff to Konnect". "KIC in Konnect" seemed to feature focused instead of use case focused.